docs/artifactory/installtransfer/help.go

```diff
@@ -0,0 +1,12 @@
+package installtransfer
+
+var Usage = []string{"rt transfer-install <server-id> [command options]"}
+
+func GetDescription() string {
+	return "Download and install the data-transfer user plugin."
+}
+
+func GetArguments() string {
+	return `	server-id
+		The source server ID. The plugin will be install on the primary node of this server.`
```

Comment:
```suggestion
		The source server ID. The plugin will be installed on the primary node of this server.`
```
Also - It is not clear if `The plugin will be installed on the primary node of this server` is a description or a precondition. If it is a precondition, it shouldn't be in this flag, but in the command description.

Reply:
done, moved to description